packages/customize-widgets/src/components/sidebar-block-editor/index.js

```diff
@@ -87,6 +139,24 @@ export default function SidebarBlockEditor( {
 		keepCaretInsideBlock,
 	] );
 
+	// The top of the editor header is offset by the Customizer’s section meta
+	// which changes according to scroll position and the top of the block
+	// toolbars are offset by the header.
+	const sectionMetaHeight = sectionMeta.offsetHeight;
+	const scrollback = useScrollback( scrollingContext, sectionMetaHeight );
+	const [ headerHeight, updateHeaderHeight ] = useReducer(
+		( height, { offsetHeight } ) => offsetHeight,
+		0
+	);
+	const headerRef = useRefEffect( updateHeaderHeight, [] );
```

Comment:
The header height doesn't vary so perhaps we should just hardcode it and eliminate this and the need for `Header` to forward a ref.